mongo/integration/collection_test.go

```diff
@@ -1861,6 +1863,334 @@ func TestCollection(t *testing.T) {
 	})
 }
 
+func TestBypassEmptyTsReplacement(t *testing.T) {
+	mt := mtest.New(t, mtest.NewOptions().CreateClient(false).MinServerVersion("5.0"))
+
+	marshalValue := func(val interface{}) bson.RawValue {
+		t.Helper()
+
+		valType, data, err := bson.MarshalValue(val)
+		assert.Nil(t, err, "MarshalValue error: %v", err)
```

Comment:
```suggestion
		require.NoError(t, err, "MarshalValue error: %v", err)
```

Suggest stopping test execution when we get an unexpected error.